pex/bin/pex.py

```diff
@@ -444,6 +445,17 @@ def configure_clp_pex_environment(parser):
            'string representing the python version (e.g. "27", "36"). ABI is the ABI tag '
            '(e.g. "cp36m", "cp27mu", "abi3", "none"). Default: current platform.')
 
+  group.add_option(
+      '--resolve-local-platforms',
+      dest='resolve_local_platforms',
+      default=False,
+      action='callback',
+      callback=parse_bool,
+      help='When --platforms are specified, attempt to resolve a local interpreter that matches '
+           'each platform specified. If found, use the interpreter to resolve distributions; if '
+           'not, resolve for the platform only allowing matching binary distributions and failing '
```

Comment:
bdist = built distribution, not binary.

```suggestion
           'not, resolve for the platform only allowing matching built distributions (bdists) and failing '
```

Reply:
There are a couple other references in this PR that would need updating.

Reply:
Well, you're running into reality here. Pip, which is what we delegate all this to uses binary == built:
https://github.com/pantsbuild/pex/blob/dd0cf31f791cfce77ef29ddc0750985f80135850/pex/pip.py#L219-L223

Reply:
Hm, that's confusing. Distutils calls it a built distribution: https://docs.python.org/3/distutils/builtdist.html, but that's fine to keep using this if you prefer.

Reply:
Yeah, I think we had talked about distinction without a difference. I don't think its long term useful to be hung up on this one. Unless there is a large change, bdist == binary == {wheel}. Once egg was part of the set, but not really any longer.